.github/workflows/ci.yml

```
@@ -0,0 +1,14 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-16.04
```

```suggestion
    runs-on: ubuntu-18.04
```

Any particular reason for using old ubuntu?
Looks like 18.04 is also supported: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/virtual-environments-for-github-hosted-runners

---

Just copied it from the tutorial.